TodayView/TodayViewController.swift

```diff
@@ -9,11 +9,12 @@
 
 import UIKit
 import NotificationCenter
+import OBAKit
```

Comment:
This won't work:

```
ld: warning: linking against a dylib which is not safe for use in application extensions: /Users/aaron/Library/Developer/Xcode/DerivedData/OBAKit-gelmlmyzyvyfksgmckeavpuczvmc/Build/Products/Debug-iphonesimulator/OBAKit.framework/OBAKit
```

You'll need to move the functionality that you're using in OBAKit into OBAKitCore or something similar.

Reply:
- Revert TodayExtension → OBAListView

TodayView depends on OBAKit for OBAListView. Since TodayView is for iOS 13 only, I don't see the need to move OBAListView to OBAKitCore just for that functionality.
Probably should rewrite TodayView without IGListKit or porting TodayView from old-oba (which uses stack views).